lambda/MAS.Tests/IntergrationTests/MailControllerTests.cs

```diff
@@ -28,6 +30,7 @@ public async Task CreateAndSendCampaign()
             //Act
             var response = await _client.PutAsync("/api/mail/daily", null);
 
+            //Get campaign to check if it saved
             var campaignId = await response.Content.ReadAsStringAsync();
             var campaign = await client.GetAsync("https://us5.api.mailchimp.com/3.0/campaigns/" + campaignId);
```

Comment:
Should we set the base url up `"https://us5.api.mailchimp.com/3.0/campaigns/` as constant so it can be reused.

Reply:
Done